model/problem.py

```diff
@@ -59,6 +59,9 @@ def view_problem_list(user, offset, count, problem_id, tags, name, course):
                     'submitter': p.submitter,
                     'tags': p.tags,
                     'type': p.problem_type,
+                    'quota': p.quota,
+                    'submitCount': user.problem_submission.get(
```

Comment:
submitCount for what?

Reply:
a player's submission count for that problem in that day

Reply:
so, all the information is stroed in problem?

Reply:
no, submission count is stored in User